js/common.js

```diff
@@ -1733,6 +1734,12 @@ var App = new function() {
             }
         }
     };
+        NoteView.hideDoneButton = function(){
+            elSave.classList.remove(CLASS_WHEN_VISIBLE);
+        };
+        NoteView.showDoneButton = function(){
+            elSave.classList.add(CLASS_WHEN_VISIBLE);
+        };
```

Comment:
this block is in fact in the wrong place. this is referencing a variable that it doesn't have access in this scope (variable is declared inside the `var NoteView = new function() { ... }`)